### ISSUE TITLE
IMGS_PER_GPU setting

### ISSUE THREAD

**Issue by RikkyLai** [sunnyHelen/JPerceiver#7]
According to the original config setting, the `IMGS_PER_GPU` is set to 3.  However, the GPUs I use now are GTX 3080 10 GB so I changed the `IMGS_PER_GPU` to 1. Up to 120 epochs training (kitti odom), the _eval_ results are 
`abs_rel: 0.2068, sq_rel: 1.5355, rmse: 4.9815, rmse_log: 0.2898, a1: 0.7079, a2: 0.9595, scale_mean: 1.7448, iou_road: 0.7133, mAP_road: 0.8781`
These results have been achieved for around 50 epochs and haven't shown any descending trends.
Does the `IMGS_PER_GPU` set to 1 have a negative effect on the training/eval procedure?

**Comment by sunnyHelen** [comment]
Yes, the batch size is an important parameter for depth estimation tasks. A smaller batch size might affect the performance but the affection shouldn't be so large. BTW, if you changed the batch size, did you change the learning rate accordingly?

**Comment by RikkyLai** [comment]
Yes, the original learning rate was 1e-4, and after 50 epochs the learning rate was changed to 1e-5. Since the results didn't show any descending trends, I changed the learning rate to 1e-6 after 100 epochs.

**Comment by sunnyHelen** [comment]
I'm not saying the learning changes with the training epoch. I mean you changed the default batch size, then you should change the default learning rate accordingly.

**Comment by RikkyLai** [comment]
Thanks for your reply. My GPU devices are two GTX 3080 10 GB, _batch size_ is changed to 1, could you give me advice about how to set the learning rate?
